modules/seqexec/server/src/main/scala/seqexec/server/keywords/package.scala

```diff
@@ -237,26 +238,36 @@ package object keywords {
   }
 
   implicit class SeqActionOption2SeqActionF[F[_]: Functor, A: DefaultHeaderValue](
-      val v: SeqActionF[F, Option[A]]) {
-    def orDefault: SeqActionF[F, A] = v.map(_.orDefault)
+      val v: F[Option[A]]) {
+    def orDefault: F[A] = v.map(_.orDefault)
   }
 
-  def buildKeyword[F[_]: Functor, A](get: SeqActionF[F, A], name: KeywordName, f: (KeywordName, A) => Keyword[A]): KeywordBag => SeqActionF[F, KeywordBag] =
+  def buildKeywordF[F[_]: Functor, A](get: F[A], name: KeywordName, f: (KeywordName, A) => Keyword[A]): KeywordBag => F[KeywordBag] =
     k => get.map(x => k.add(f(name, x)))
-  def buildInt8[F[_]: Functor](get: SeqActionF[F, Byte], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag]       = buildKeyword(get, name, Int8Keyword)
-  def buildInt16[F[_]: Functor](get: SeqActionF[F, Short], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag]     = buildKeyword(get, name, Int16Keyword)
-  def buildInt32[F[_]: Functor](get: SeqActionF[F, Int], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag]       = buildKeyword(get, name, Int32Keyword)
-  def buildFloat[F[_]: Functor](get: SeqActionF[F, Float], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag]     = buildKeyword(get, name, FloatKeyword)
-  def buildDouble[F[_]: Functor](get: SeqActionF[F, Double], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag]   = buildKeyword(get, name, DoubleKeyword)
-  def buildBoolean[F[_]: Functor](get: SeqActionF[F, Boolean], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag] = buildKeyword(get, name, BooleanKeyword)
-  def buildString[F[_]: Functor](get: SeqActionF[F, String], name: KeywordName): KeywordBag => SeqActionF[F, KeywordBag]   = buildKeyword(get, name, StringKeyword)
-
-  def sendKeywords[F[_]: Sync](
+  def buildKeyword[F[_]: MonadError[?[_], Throwable], A](get: SeqActionF[F, A], name: KeywordName, f: (KeywordName, A) => Keyword[A]): KeywordBag => F[KeywordBag] =
```

Comment:
We keep two set of functions to support the existing headers built around `SeqActionF`.
Should disappear at some moment